### PR TITLE
fix: sender name in mandrill provider

### LINKED ISSUE #4558: 🐛 Bug Report: Missing Sender Name in Mandrill (Email) Provider
### 📜 Description

`Sender Name` doesn't work in Mandrill Provider, as the [current implementation](https://github.com/novuhq/novu/blob/next/providers/mandrill/src/lib/mandrill.provider.ts#L46-L58) is missing the `from_name` parameter [required by Mandrill](https://mailchimp.com/developer/transactional/api/messages/send-new-message/) to set up the sender's name in the email.

### 👟 Reproduction steps

1.  Setup Mandrill provider in Novu Cloud
2. Create a workflow and implement a email template with Mandrill provider
3. Send a test email and check it in your inbox.
4. You might find there's no sender name.

### 👍 Expected behavior

`SenderName <email@domain.com>` is the expected value in the `from` field in received email

### 👎 Actual Behavior with Screenshots

Actually will only receive `email@domain.com`, although you have set up the `Sender Name` in Novu.

### Novu version

0.20.0

### npm version

_No response_

### node version

_No response_

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by emersonliuuu**
### What change does this PR introduce?

Fixed issue #4558 by implementing an additional parameter that Mandrill required. 
### Why was this change needed?

Closes #4558

### CODE REVIEW

File: providers/mandrill/src/lib/mandrill.provider.spec.ts
Comment (p-fernandez) [inline providers/mandrill/src/lib/mandrill.provider.spec.ts:1]: You need to fix this test. In line 37 you need to add the check for `from_name` with the expectad value you gave in the `mockConfig` in line 6.

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Thank you for your contribution.
Once fixed the failing test this PR is good to go!

File: providers/mandrill/src/lib/mandrill.provider.spec.ts
Comment (emersonliuuu) [reply providers/mandrill/src/lib/mandrill.provider.spec.ts:1]: Thanks for pointing out the missing part!

**Review by p-fernandez — APPROVED** [review]
Thank you for your contribution. 🌟

**Pull request merged**